index.js

```diff
@@ -27,12 +41,16 @@ class Timber {
 		}
 	}
 
+	get options() {
```

Comment:
Let prefix this with `_`. I don't see why anyone would need this. We can rather expose it if there's an actual need.

Reply:
The property `this._options` is already taken by the actual object on the instance (not the getter method), that's why I originally left it as `this.options`. Should we add an extra `_` to both? Or rename one/both?

Reply:
You could rename the current `this._options` to `this._initialOptions`